shared/config.gradle

```diff
@@ -20,6 +20,8 @@ nativeUtils.wpi.configureDependencies {
 nativeUtils.wpi.addWarnings()
 nativeUtils.wpi.addWarningsAsErrors()
 
+nativeUtils.setSinglePrintPerPlatform()
```

Comment:
What's this do?

Reply:
No clue, copied it from vendor template, not sure if we need it.